lib/jekyll/theme.rb

```diff
@@ -63,7 +63,8 @@ def realpath_for(folder)
       # However, symlinks are allowed to point to other directories within the theme.
       Jekyll.sanitized_path(root, File.realpath(Jekyll.sanitized_path(root, folder.to_s)))
     rescue Errno::ENOENT, Errno::EACCES, Errno::ELOOP
-      Jekyll.logger.warn "Invalid theme folder:", folder
+      Jekyll.logger.warn "Path '#{folder}' does not exist, is not accessible "\
```

Comment:
Since we're aiming for clarity here, lets be verbose.. Split into two logger calls for shorter readable msgs:
```
Warning: The '#{folder}' directory does not exist at the theme's root,
         is either not accessible or includes a symbolic link loop
```

Reply:
```suggestion
      Jekyll.logger.warn "Warning:", "The '#{folder}' directory does not exist at theme's root,"
      Jekyll.logger.warn "", "is either not accessible or includes a symbolic link loop"
```

Reply:
Yes, I see what you mean.

With the two logger calls the message properly aligns with the rest of the log:

```
      Generating... 
           Warning: The '_sass' directory does not exist at theme's root,
                    is either not accessible or includes a symbolic link loop
      Remote Theme: Using theme ...
```

👍